libs/cardano-ledger-api/cardano-ledger-api.cabal

```diff
@@ -36,6 +36,7 @@ library
         Cardano.Ledger.Api.Tx.In
         Cardano.Ledger.Api.Tx.Out
         Cardano.Ledger.Api.Tx.Wits
+        Cardano.Ledger.Api.State.CommitteeMembersState
```

Comment:
This should be a hidden module with all types re-exported from `Cardano.Ledger.Api.State.Query`